src/main.py

```
@@ -1,373 +1,19 @@
-            )
-        ),
-    )
-
+app.include_router(factcheck_router)
 
 if __name__ == "__main__":
     import uvicorn
-
-    logger.info("Starting VeriFact API server")
-    uvicorn.run(app, host="0.0.0.0", port=8000)
+    uvicorn.run(app, host="0.0.0.0", port=8000)
```

Ruff (0.11.9)

No newline at end of file

Add trailing newline

(W292)